src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -3,15 +3,18 @@ import { Validators } from 'react-reactive-form';
 
 import IncidentNavigation from '../components/IncidentNavigation';
 import FormComponents from '../components/form';
-import { checkVisibility } from '../services/format-conditional-form';
+import checkVisibility from '../services/check-visibility';
 
 export default {
   label: 'Beschrijf uw melding',
-  getNextStep: (wizard, { subcategory, category }) => {
-    if (!some(wizard.vulaan.form.controls, (control) => control.meta && control.meta.pathMerge && checkVisibility(control, {
-      category,
-      subcategory
-    }))) {
+  getNextStep: (wizard, incident, isAuthenticated) => {
```

Comment:
getNextStep returns bool instead of `Step`. Probaby change the name so it is clear that a bool is returned.